src/main/java/frc/robot/subsystems/TriggerSubsystem.java

```diff
@@ -3,15 +3,17 @@
 import com.revrobotics.CANSparkLowLevel.MotorType;
 import com.revrobotics.CANSparkMax;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import edu.wpi.first.wpilibj2.command.Subsystem;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.TriggerConstants;
 
 public class TriggerSubsystem extends SubsystemBase {
   private CANSparkMax motor;
 
   public TriggerSubsystem() {
-    motor = new CANSparkMax(TriggerConstants.MOTOR_ID, MotorType.kBrushless);
+    motor = new CANSparkMax(TriggerConstants.MOTOR_ID, MotorType.kBrushed);
```

Comment:
Annoyingly there is a bug that causes simulation crashes with kBrushed motors. 
[[BUG]: Creating a brushed Spark Max aborts in sim](https://github.com/robotpy/robotpy-rev/issues/55) 

Looks like there is new REVLib update that fixes this.

Reply:
Resolved in ecb475f